kasa/smart/effects.py

```diff
@@ -2,8 +2,33 @@
 
 from __future__ import annotations
 
+from abc import ABC, abstractmethod
 from typing import cast
 
+from ..interfaces.lighteffect import LightEffect as LightEffectInterface
+
+
+class SmartLightEffect(LightEffectInterface, ABC):
```

Comment:
This could perhaps be stored inside `smart.interfaces` module?

Reply:
We don't currently have a `smart.interfaces` module, maybe it could be added if we ever need more than one. As it's internal I think this is a good enough place for now.